packages/lib/shared/components/tooltips/apr-tooltip/BaseAprTooltip.tsx

```diff
@@ -161,34 +170,44 @@ function BaseAprTooltip({
       shadow="3xl"
       w="fit-content"
     >
-      <TooltipAprItem
-        {...basePopoverAprItemProps}
-        apr={swapFeesDisplayed}
-        aprOpacity={isSwapFeePresent ? 1 : 0.5}
-        displayValueFormatter={usedDisplayValueFormatter}
-        pt={3}
-        title="Swap fees"
-        tooltipText={swapFeesTooltipText}
-      >
-        {hookType ? (
-          <>
-            <TooltipAprItem
-              {...subitemPopoverAprItemProps}
-              apr={bn(swapFeesDisplayed).minus(dynamicSwapFeesDisplayed)}
-              displayValueFormatter={usedDisplayValueFormatter}
-              title="Regular swap fees"
-            />
-            <TooltipAprItem
-              {...subitemPopoverAprItemProps}
-              apr={dynamicSwapFeesDisplayed}
-              displayValueFormatter={usedDisplayValueFormatter}
-              title={getDynamicSwapFeesLabel(hookType)}
-              tooltipText={dynamicSwapFeesTooltipText[hookType as SupportedHookType]}
-            />
-          </>
-        ) : null}
-      </TooltipAprItem>
-
+      {isQuantAmm ? (
```

Comment:
Does quantAMM not have swap fees?

Reply:
No, the swap fees are somewhat accounted for in the quant amm apr.